lib/internal/http.js

```diff
@@ -1,20 +1,21 @@
 'use strict';
 
-const timers = require('timers');
+const { setUnrefTimeout } = require('internal/timers');
 
 var dateCache;
 function utcDate() {
   if (!dateCache) {
     const d = new Date();
     dateCache = d.toUTCString();
-    timers.enroll(utcDate, 1000 - d.getMilliseconds());
-    timers._unrefActive(utcDate);
+
+    setUnrefTimeout(timeout, 1000 - d.getMilliseconds());
   }
   return dateCache;
 }
-utcDate._onTimeout = function() {
+
+function timeout() {
```

Comment:
Maybe call this function `resetCache` or sth like that, `timeout` doesn't really say what it's doing...